node/npm/glide/index.ts

```diff
@@ -85,6 +106,13 @@ function initialize() {
         ConnectionError,
         ClusterTransaction,
         Transaction,
+        createLeakedArray,
+        createLeakedAttribute,
+        createLeakedBigint,
+        createLeakedDouble,
+        createLeakedMap,
+        createLeakedString,
+        parseInfoResponse,
     } = nativeBinding;
 
     module.exports = {
```

Comment:
can't you do:
```
module.exports = nativeBinding; 
```

Reply:
Its out of the scope of the PR, the additions are just for the CI to pass as it missed.
I can try in another PR but i think the reason its built like that is to support both JS styles - ES and CJS.